scripts/prepare_lambda_deployment.py

```diff
@@ -39,10 +39,8 @@ def transform_tf(input_json):
     }
 
     for func in input_json['resource']['aws_lambda_function'].values():
-        assert func['source_code_hash'] == '${filebase64sha256("./deployment.zip")}'
-        func['source_code_hash'] = '${filebase64sha256("${path.module}/deployment.zip")}'
-        assert func['filename'] == "./deployment.zip"
-        func['filename'] = "${path.module}/deployment.zip"
+        assert func['source_code_hash'] == '${filebase64sha256("${path.module}/deployment.zip")}', func
+        assert func['filename'] == "${path.module}/deployment.zip"
```

Comment:
I think it's safe to remove these asserts. I only added them initially so that we would remember to remove the modifications to the generated TF config once the chalice bugs were fixed.